engines/stark/ui/world/gamewindow.cpp

```diff
@@ -117,7 +117,6 @@ void GameWindow::onRender() {
 
 void GameWindow::onMouseMove(const Common::Point &pos) {
 	_renderEntries = StarkGlobal->getCurrent()->getLocation()->listRenderEntries();
-	_cursor->setFading(false);
```

Comment:
Since setting the flag has a side effect of controlling sound playback now, we should avoid setting it back and forth to prevent playback artifacts.